src/components/playback/playbackmanager.js

```diff
@@ -1,729 +1,738 @@
-                }
+        // media source is in network, but connection is out of network
+        return Promise.resolve(false);
+    });
+}
 
-                return Promise.resolve(true);
-            }
+function supportsDirectPlay(apiClient, item, mediaSource) {
+    // folder rip hacks due to not yet being supported by the stream building engine
+    const isFolderRip = mediaSource.VideoType === 'BluRay' || mediaSource.VideoType === 'Dvd' || mediaSource.VideoType === 'HdDvd';
 
-            // media source is in network, but connection is out of network
+    if (mediaSource.SupportsDirectPlay || isFolderRip) {
+        if (mediaSource.IsRemote && !apphost.supports('remotevideo')) {
             return Promise.resolve(false);
-        });
-    }
-
-    function supportsDirectPlay(apiClient, item, mediaSource) {
-        // folder rip hacks due to not yet being supported by the stream building engine
-        var isFolderRip = mediaSource.VideoType === 'BluRay' || mediaSource.VideoType === 'Dvd' || mediaSource.VideoType === 'HdDvd';
+        }
 
-        if (mediaSource.SupportsDirectPlay || isFolderRip) {
-            if (mediaSource.IsRemote && !apphost.supports('remotevideo')) {
-                return Promise.resolve(false);
+        if (mediaSource.Protocol === 'Http' && !mediaSource.RequiredHttpHeaders.length) {
+            // If this is the only way it can be played, then allow it
+            if (!mediaSource.SupportsDirectStream && !mediaSource.SupportsTranscoding) {
+                return Promise.resolve(true);
+            } else {
+                return isHostReachable(mediaSource, apiClient);
             }
-
-            if (mediaSource.Protocol === 'Http' && !mediaSource.RequiredHttpHeaders.length) {
-                // If this is the only way it can be played, then allow it
-                if (!mediaSource.SupportsDirectStream && !mediaSource.SupportsTranscoding) {
-                    return Promise.resolve(true);
-                } else {
-                    return isHostReachable(mediaSource, apiClient);
-                }
-            } else if (mediaSource.Protocol === 'File') {
-                return new Promise(function (resolve, reject) {
-                    // Determine if the file can be accessed directly
-                    require(['filesystem'], function (filesystem) {
-                        var method = isFolderRip ?
-                            'directoryExists' :
-                            'fileExists';
-
-                        filesystem[method](mediaSource.Path).then(function () {
-                            resolve(true);
-                        }, function () {
-                            resolve(false);
-                        });
+        } else if (mediaSource.Protocol === 'File') {
+            return new Promise(function (resolve, reject) {
+                // Determine if the file can be accessed directly
+                import('filesystem').then(({ default: filesystem }) => {
```

Comment:
`filesystem` module has no `default` export.
```suggestion
                import('filesystem').then((filesystem) => {
```

Reply:
@Camc314 Did you forget to pull a branch after GH commits? Re-apply this one.

Reply:
Yep. Not again :p